### DISCUSSION TITLE
Enable italics for only comments?

### DISCUSSION THREAD

**Discussion by iton0** [rmehri01/onenord.nvim#84] [Q&A]
Currently, the selected line in the quickfix list and some of the descriptions in WhichKey are italics. I want to have italics disabled globally for the colorscheme, but be able to enable them for just comments. Also is there a way to disable underline globally as well, in particular with the yank highlighting.

**Comment by rmehri01** [comment]
Hey, I think you should be able to overwrite `WhichKeyDesc` to not have italics and then add them only to `Comment` and `@comment` using `custom_highlights`. For disabling underline do you mean the `LspReference*` groups or something else?

**Reply by iton0** [reply]
It was for IncSearch highlight but I was able to resolve it earlier today. Thank you for responding and thank you for the awesome colorscheme!

**Reply by rmehri01** [reply]
Awesome! Glad you like the theme 😄
